index.js

```diff
@@ -4,11 +4,15 @@ var chalk = require('chalk');
 var inquirer = require('inquirer');
 var axios = require('axios');
 var cheerio = require('cheerio');
+var NodeCache = require("file-system-cache").default;
 
 var baseUrl = 'http://kbbi4.portalbahasa.com';
 var arti = [];
 var kataBerimbuhan = [];
 var gabunganKata = [];
+var myCache = NodeCache();
```

Comment:
Ah right, I didnt follow previous style. Noted

On Aug 3, 2016 09:41, "Muhammad Adinata" <notifications@github.com> wrote:

> In index.js
> <https://github.com/aliakbars/kbbi/pull/1#discussion_r73272939>:
>
> >
> >  var baseUrl = 'http://kbbi4.portalbahasa.com';
> >  var arti = [];
> >  var kataBerimbuhan = [];
> >  var gabunganKata = [];
> > +var myCache = NodeCache();
>
> Should have consistent style, either snake_case or camelCase
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/aliakbars/kbbi/pull/1/files/789eb54c480c353ffa22be4b1dd106f1f2f8644d#r73272939>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADx0EC3rKPpjmQpLC5oq4kzoSOkMDhKgks5qb__UgaJpZM4JZTTI>
> .
>